source/NPC.cpp

```diff
@@ -72,17 +72,16 @@ void NPC::Load(const DataNode &node)
 		else if(node.Token(i) == "evade")
 			mustEvade = true;
 		else if(node.Token(i) == "accompany")
-		{
 			mustAccompany = true;
-			failIf |= ShipEvent::DESTROY;
-		}
 		else
 			node.PrintTrace("Warning: Skipping unrecognized NPC completion condition \"" + node.Token(i) + "\":");
 	}
 
 	// Check for incorrect objective combinations.
 	if(failIf & ShipEvent::DESTROY && (succeedIf & ShipEvent::DESTROY || succeedIf & ShipEvent::CAPTURE))
 		node.PrintTrace("Error: conflicting NPC mission objective to save and destroy or capture.");
+	if(mustEvade && mustAccompany)
+		node.PrintTrace("Error: conflicting NPC mission objective to accompany and evade.");
```

Comment:
This was true, but it is no longer true with this PR.
If you can complete an `accompany` objective by letting the ships all die, then `accompany evade` is a perfectly possible objective. It'd be a bit weird to do that instead of just `kill`, but it's not a problem.

Reply:
`accompany evade` is either strictly conflicting (you can't succeed by accompanying the ship to the destination because you aren't evading it) or redundant (accompany and evade can both be "succeeded" by killing the ship). In either case, the combination of the two should be warned about.

Reply:
You should call it redundant, not conflicting, then.

Reply:
This good?
```suggestion
	if(mustEvade && mustAccompany)
		node.PrintTrace("Warning: NPC mission objective to accompany and evade is synonymous with kill.");
```